CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+Any contribution that you make to this repository will
+be under the BSD license 2.0, as dictated by that
```

Comment:
You list BSD2 but link is BSD3. 

Also I might ask that if we’re going to be specific, we allow for Apache 2.0. If I can’t contribute back in Apache, then I’m not able to help maintain during my work time, and this is true for Amazon and other larger open-source companies.

Reply:
All of the files in this repo have been under BSD from the beginning. I'm fine with new contributions being under Apache 2.0 but, if we want to change the license for any of the existing code, we would need to contact the original developers to get approval. We could also dual-license but we'll need to be careful about how we handle that. For now, I'll change the CONTRIBUTING.md to point to Apache.

Regarding the BSD 2/3 - The original or "simplified" BSD license is the "2-clause" license and the "extended" BSD license is the "3-clause," sometimes also called BSD 2.0.

Reply:
According to http://www.apache.org/legal/resolved.html#category-a, BSD 3-clause code can be included in an Apache 2.0-licensed project. How would you like to handle this?